### PR TITLE
remove redis token persistence backend

### LINKED ISSUE #8932: Remove `RedisTokenPersistence`
:clipboard: **Description**

Lets leave it in for the March and April releases then after we've done the April release, remove it so it is gone in the May release.

@SnoozeThis wait until April 2 2023

### PR DESCRIPTION
**Pull request by chris48s**
closes #8932

When I assemble the changelog for the May release, I'll call this out at the top.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -42,15 +42,6 @@ if (fs.existsSync('.env')) {
   process.exit(1)
 }
 
-if (config.private.redis_url != null) {
```
Comment (chris48s) [inline server.js:45]: The decision I have taken here is just to remove `redis_url` from the config schema.
The other thing we could do would be leave it in the schema, keep this line here and throw an exception if `redis_url` is set.

Comment (calebcartwright) [reply server.js:45]: Long term I don't think it's necessary to have that, though I find myself somewhat ambivalent about what to do in the short term. I could see value in having the exception explicitly thrown for another monthly release or two

**Review by calebcartwright — APPROVED** [review]
lgtm. i'm of two minds about whether to throw the exception if the var is set, so will approve in case you decide to stick with this but happy to re-:+1: if you go the other route instead

**Comment by chris48s** [comment]
belt and braces..

**Comment by chris48s** [comment]
I'm going to go ahead and merge this one

**Pull request merged**